include/IMesh.h

```diff
@@ -42,6 +42,9 @@ namespace scene
 		virtual ~IMesh() {}
 	public:
 
+		//! Resets bounding box and removes all meshbuffers (calls drop() on each).
+		virtual void clearMeshBuffers() = 0;

```

Comment:
don't add mutability to ICPUMesh API, if you have code that requires mutability and manipulation then create an SCPUMesh (or skinned mesh) and cast it when you're done